clusterloader2/pkg/prometheus/manifests/prometheus-additional-scrape-configs.yaml

```diff
@@ -0,0 +1,12 @@
+{{$WINDOWS_NODE_NAME := DefaultParam .WINDOWS_NODE_NAME ""}}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: monitoring
+  name: windows-scrape-configs
+stringData:
+  prometheus-additional.yaml: |-
```

Comment:
windows-additional.yaml? windows-scrape-configs.yaml?

To future-proof this in case someone else wants to add additional scrape configs in the future that might not apply to Windows nodes.

Reply:
make sense. changed to windows-scrape-configs.yaml